RestApi/RestApi.csproj

```diff
@@ -21,25 +21,29 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Antlr4.Runtime.Standard" Version="4.7.1.1" />
+    <PackageReference Include="Antlr4.Runtime" Version="4.6.4" />
```

Comment:
:memo: The separation into `Antlr4.Runtime` and `Antlr4.CodeGenerator` is important for this project because a NuGet package is produced directly from this project. We want the former listed as a dependency, but not the latter.